pkg/fleet/installer/setup/defaultscript/default_script.go

```diff
@@ -0,0 +1,178 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package defaultscript contains default standard installation logic
+package defaultscript
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/fleet/installer/env"
+	"github.com/DataDog/datadog-agent/pkg/fleet/installer/setup/common"
+)
+
+const (
+	defaultAgentVersion    = "7.60.1-1"
+	defaultInjectorVersion = "0.26.0-1"
+)
+
+var (
+	defaultLibraryVersions = map[string]string{
+		common.DatadogAPMLibraryJavaPackage:   "1.44.1-1",
+		common.DatadogAPMLibraryRubyPackage:   "2.8.0-1",
+		common.DatadogAPMLibraryJSPackage:     "5.30.0-1",
+		common.DatadogAPMLibraryDotNetPackage: "3.7.0-1",
+		common.DatadogAPMLibraryPythonPackage: "2.9.2-1",
+		common.DatadogAPMLibraryPHPPackage:    "1.5.1-1",
+	}
+
+	fullSemverRe = regexp.MustCompile(`^[0-9]+\.[0-9]+\.[0-9]+`)
+
+	// unsupportedEnvVars are the environment variables that are not supported by the default script
+	unsupportedEnvVars = []string{
+		"DD_INSTALLER",
+		"DD_AGENT_FLAVOR",
+		"DD_HOST_TAGS",
+		"DD_UPGRADE",
+		"DD_APM_INSTRUMENTATION_NO_CONFIG_CHANGE",
+		"DD_INSTALL_ONLY",
```

Comment:
also `DD_FIPS_MODE`, I think it was a good pattern to fail the script below for it, applying it to all env vars not supported will avoid the surprise of not working as intended